packages/CakePHP/tests/Rector/MethodCall/ModalToGetSetRector/Source/SomeModelType.php

```diff
@@ -4,7 +4,13 @@
 
 namespace Rector\CakePHP\Tests\Rector\MethodCall\ModalToGetSetRector\Source;
 
-final class SomeModelType
+class Entity
 {
+}
 
+final class SomeModelType
```

Comment:
These should be 2 separated files, as that's `/Source/`

Reply:
Fixed.